spec/models/alchemy/element_spec.rb

```diff
@@ -435,24 +435,38 @@ module Alchemy
         end
       end
 
-      context 'with nestable elements' do
-        let(:element) { build_stubbed(:alchemy_element, :with_nestable_elements, { create_contents_after_create: true } ) }
+      context 'with nested elements' do
+        let(:element) do
+          build_stubbed(:alchemy_element, :with_nestable_elements)
+        end
+
+        let(:nested_element) do
+          build_stubbed(:alchemy_element, name: 'slide')
+        end
+
         let(:preview_contents) { [preview_content] }
-        before { element.nested_elements << create(:alchemy_element, name: 'slide') }
+
+        before do
+          allow(element).to receive(:nested_elements) { [nested_element] }
+        end
 
         context 'where parent element has content marked as preview' do
-          before { allow(element).to receive(:contents).and_return(preview_contents) }
+          before do
+            allow(element).to receive(:contents).and_return(preview_contents)
+          end
 
           it 'should return the preview text for the parent element' do
-            expect(element.preview_text).to eq(element.contents.first.preview_text)
+            expect(element.preview_text).to eq('Lorem')
```

Comment:
Can you test for a different text in the parent element than in the nested element? This way, there's a danger of a false positive test because it'll be green even if the wrong element's preview text is selected.